### PR TITLE
add: Czech/Slovak Classification Datasets

### PR DESCRIPTION
**Pull request by mrshu**
## Checklist for adding MMTEB dataset

Reason for dataset addition:

This adds a Czech and a Slovak Classification dataset/task. For Slovak, this seems to be the first dataset for the Classification task.

- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [x] I have added points for my submission to the [points folder](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) using the PR number as the filename (e.g. `438.jsonl`).

### CODE REVIEW

**Review by x-tabdeveloping — APPROVED** [review]
Looks pretty good! Add points then it's pretty much ready to merge.

**Comment by mrshu** [comment]
Thanks @x-tabdeveloping -- I would appreciate if you could take another look.

**Review by x-tabdeveloping — COMMENTED** [review]
Looks good! Strangely enough it seems like a very hard task (neither of the models aren't performing well and there's virtually no gap between them). Could you perhaps try to bump up the number of training examples per class and see what happens?

**Comment by x-tabdeveloping** [comment]
I really appreciate the clean tree btw, sparks joy :D

**Comment by mrshu** [comment]
> Could you perhaps try to bump up the number of training examples per class and see what happens?

@x-tabdeveloping would you mind sharing how? I am not really sure where to look that up. Thanks!

**Comment by x-tabdeveloping** [comment]
Sure thing @mrshu . I'll also have to look it up, so gimme a sec :D The secret is to look in the AbsTasks' code.

**Comment by x-tabdeveloping** [comment]
It's very hacky but currently this is the canonical way to do it:
```python
class ThisandThat(AbsTaskClassification):
    ...

    @property
    def metadata_dict(self):
        md = super().metadata_dict
        md["samples_per_label"] = 20 # or whatever number
        return md
```

**Comment by mrshu** [comment]
Thanks @x-tabdeveloping! I did that and the results have improved a bit (by roughly 0.03 F1 points in absolute terms) so it did help and the performance doesn't seem to be random, but it's far from what you'd expect on a comparable English dataset for instance.

Unfortunately, my suspicion is that this says reflects the current quality of multilingual embeddings in Czech/Slovak contexts, which is also why I was hoping to include these tasks in the benchmark :slightly_smiling_face: 

Let me know if there are other things I should update!

File: mteb/tasks/Classification/ces/CSFDCZMovieReviewSentimentClassification.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        type="Classification",
+        category="s2s",
+        date=("2002-06-28", "2020-03-13"),
+        eval_splits=["test"],
+        eval_langs=["ces-Latn"],
+        main_score="accuracy",
+        form=["written"],
+        domains=["Reviews"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license="CC-BY-SA-4.0",
```
Comment (KranthiGV) [inline mteb/tasks/Classification/ces/CSFDCZMovieReviewSentimentClassification.py:27]: Where did we get this license info from?

File: mteb/tasks/Classification/ces/CSFDCZMovieReviewSentimentClassification.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        socioeconomic_status="mixed",
+        annotations_creators="derived",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation="""
+@misc{štefánik2023resources,
+      title={Resources and Few-shot Learners for In-context Learning in Slavic Languages},
```
Comment (KranthiGV) [inline mteb/tasks/Classification/ces/CSFDCZMovieReviewSentimentClassification.py:34]: Is this the correct citation?
The paper doesn't seem to mention this dataset

File: mteb/tasks/Classification/ces/CSFDCZMovieReviewSentimentClassification.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+}
+""",
+        n_samples={"test": N_SAMPLES},
+        avg_character_length={"test": 386.5},
+    )
+
+    @property
+    def metadata_dict(self):
+        md = super().metadata_dict
+        md["samples_per_label"] = 20
```
Comment (KranthiGV) [inline mteb/tasks/Classification/ces/CSFDCZMovieReviewSentimentClassification.py:49]: nit: A comment on why it's modified would be great.

File: mteb/tasks/Classification/ces/CSFDCZMovieReviewSentimentClassification.py (hunk 1 above)
Comment (mrshu) [reply mteb/tasks/Classification/ces/CSFDCZMovieReviewSentimentClassification.py:27]: This is what the metadata on HuggingFace Datasets reports: https://huggingface.co/datasets/fewshot-goes-multilingual/cs_csfd-movie-reviews

image

File: mteb/tasks/Classification/ces/CSFDCZMovieReviewSentimentClassification.py (hunk 2 above)
Comment (mrshu) [reply mteb/tasks/Classification/ces/CSFDCZMovieReviewSentimentClassification.py:34]: @KranthiGV It actually does in Section 3.1.1 (page 3 of https://arxiv.org/pdf/2304.01922):

image

File: mteb/tasks/Classification/ces/CSFDCZMovieReviewSentimentClassification.py (hunk 3 above)
Comment (mrshu) [reply mteb/tasks/Classification/ces/CSFDCZMovieReviewSentimentClassification.py:50]: Thanks, added that real quick :)

**Pull request merged**